manager_utils/tests/models.py

```diff
@@ -10,6 +11,8 @@ class TestModel(models.Model):
     int_field = models.IntegerField(null=True, unique=True)
     char_field = models.CharField(max_length=128, null=True)
     float_field = models.FloatField(null=True)
+    json_field = JSONField(default=dict)
+    array_field = ArrayField(models.CharField(max_length=128), default=list)
```

Comment:
can you add a test bulk upserting / updating these two fields? or just add these fields to an existing test

Reply:
added